styles/lib/_modal.css

```diff
@@ -0,0 +1,55 @@
+.modal{
+    display: none;
+    position: fixed;
+    z-index: 100;
+    padding-top: 12rem;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+    background-color: rgba(0,0,0,0.4);
```

Comment:
try to use color variables

Reply:
I will not use this color elsewhere. That's why I haven't used variable name.